multiversx/mx-api-service
# Hot swappable settings

## Reasoning 
- Some config flags should be updateable also outside static configs, which are usually maintained inside the infrastructure settings. They should be dynamic and re-applied upon redeploy
  
## Proposed Changes
- Expose endpoint for changing certain settings which are then used when initializing the service or on-demand

## How to test
- Change values in the db for the settings. Make sure the new values are loaded after the cache expires
